### PR TITLE
maintainers: add mudrii

### PR DESCRIPTION
**Pull request by mudrii**
###### Motivation for this change

Allow user to contribute to the NixOS pakgs

###### Things done

Add new User intro the maintainers-list.nix

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by RaghavSood** [comment]
Are you intending to take maintainership of some packages? If so, you should add some commits adding yourself to the maintainer list there

Also note that just being in this file doesn't allow or stop you from contributing - you can open pull requests for any package even if you aren't listed as a maintainer.

Being listed as a maintainer for packages is just a way for the build tooling and other contributors to know who should review a pull request.

**Comment by mudrii** [comment]
Yes I will take pulumi and will separate plugins in a diff package

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -6129,6 +6129,12 @@
     githubId = 5139265;
     name = "James Wood";
   };
+  mudrii = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:6132]: Please format the commit message as like ``maintainers: add mudrii``.

**Review by mudrii — COMMENTED** [review]
title change and commit message

**Comment by mudrii** [comment]
I will close this pull request and will reinitiate new one with all fixes included

**Pull request closed without merging**